internal/linker/index.js

```diff
@@ -250,33 +269,66 @@ Include as much of the build output as you can without disclosing anything confi
             const workspaceAbs = path.resolve(workspaceDir);
             // Now add symlinks to each of our first-party packages so they appear under the node_modules tree
             const links = [];
-            const linkModule = (name, root, modulePath) => __awaiter(this, void 0, void 0, function* () {
-                let target = '<package linking failed>';
-                switch (root) {
-                    case 'bin':
-                        // FIXME(#1196)
-                        target = path.join(workspaceAbs, bin, toWorkspaceDir(modulePath));
-                        break;
-                    case 'src':
-                        target = path.join(workspaceAbs, toWorkspaceDir(modulePath));
-                        break;
-                    case 'runfiles':
-                        target = runfiles.resolve(modulePath) || '<runfiles resolution failed>';
-                        break;
+            const linkModules = (modules) => __awaiter(this, void 0, void 0, function* () {
```

Comment:
nit: prefer syntax
`function foo() {}` rather than `const foo = () => {}`
that's in the goog js/ts style guide

Reply:
👍